assets/app.less

```diff
@@ -122,3 +122,14 @@ main {
     }
 }
 
+/** Custom CSS added for Safari and IE support */
+.uls-menu{
+    z-index: 1020;
```

Comment:
Great, this looks generally good, but setting a high z-index is usually considered a bit of a hack. It's done, but it also deserves explanation (in comment, ideally) so that others who look at this code in a year or two understand why this was done.
Can you add a comment in here explaining why these are needed, and why so high?
Is there a bug in Safari/IE with the z-indexes, or is this a matter of Leaflet stealing the z-index by itself?

You can also explain it in the commit message or in the ticket, and then add a comment in the code, something like
`/* See T214937 */` -- as long as the explanation is available for posterity.